examples/model.py

```diff
@@ -193,6 +195,13 @@ class Meta:
     Thread.tags.remove()
 ]))
 
+# Update list attribute
+print(thread_item.update(actions=[
+    Thread.notes.set(
```

Comment:
Is this really how it works? You set an append action?

Reply:
Yes. You cannot use append alone. 

Here is an example in raw format:

    "UpdateExpression" : "SET #attrName = list_append(#attrName, :attrValue)",

From https://stackoverflow.com/questions/33278475/dynamodb-set-list-append-not-working-using-aws-sdk

Also, check #434

Reply:
Thanks!